packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,70 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw, HexString } from 'web3-utils';
+import { InvalidPrivateKeyError, PrivateKeyLengthError } from './errors';
+
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get account from private key
+ */
+export const privateKeyToAccount = (
+	privateKey: string | Buffer,
+): {
+	address: string;
+	privateKey: string;
+	signTransaction: () => boolean; // From 1.x
+	sign: () => boolean;
+	encrypt: () => boolean;
+} => {
+	if (!privateKey) {
+		throw new InvalidPrivateKeyError(privateKey);
+	}
+
+	const stringPrivateKey =
+		typeof privateKey === 'object' ? Buffer.from(privateKey).toString('hex') : privateKey;
+
+	const updatedKey = stringPrivateKey.startsWith('0x')
+		? stringPrivateKey.slice(2)
+		: stringPrivateKey;
+
+	// Must be 64 hex characters
+	if (updatedKey.length !== 64) {
+		throw new PrivateKeyLengthError(updatedKey);
+	}
+
+	const publicKey = getPublicKey(updatedKey);
+
+	const publicKeyString = `0x${publicKey.slice(2)}`;
```

Comment:
A comment for why `.slice(2)` is being performed would be helpful

Reply:
I'm not too sure, ether and our old library did this when creating addresses